content/en/events/upcoming-events/gsoc-2025.md

```diff
@@ -67,339 +67,177 @@ To participate in GSoC with Kubeflow, you __must__ meet the GSoC [eligibility re
-* Define CRDs, if applicable, under `crds/`.  
-* Write Deployment, Service, Ingress, and ConfigMap templates.  
-* Configure Persistence (PVCs, VolumeClaims) for storage-based components.  
-* Implement RBAC and authentication for security.
-
-###### Parameterize Configuration
-
-* Use `values.yaml` to enable modular configurations.  
-* Implement Helm templating (`{{ .Values.foo }}`) for flexibility.  
-* Provide default configurations and allow for overrides.
-
-###### Handle Dependencies (Optional)
-
-* Define external dependencies (e.g., Istio, cert-manager) in `requirements.yaml`.  
-* Evaluate subcharts for modular deployments.
-
-##### 2\. Document Helm Chart and Configuration Values
-
-* Write detailed documentation for `values.yaml` settings.  
-* Describe deployment requirements (e.g., Kubernetes version, resource limits).  
-* Include troubleshooting guidance for common issues.
-
-##### 3\. Provide a Tutorial for Deploying the Helm Chart
-
-* Select a Kubernetes distribution (e.g., Kind, Minikube, EKS, GKE, AKS).  
-* Develop a step-by-step deployment guide with verification steps.  
-* Include sample configurations for different use cases (e.g., minimal vs. full deployment).  
-* Document upgrade paths (`helm upgrade` and rollback strategies).
-
-#### Expected Outcomes:
-
-✅ A fully functional Kubeflow Helm chart for 1.9 that integrates KServe and Knative  
-✅ Published Helm chart documentation with clear configuration options  
-✅ A step-by-step tutorial that simplifies Kubeflow deployment for users  
-✅ Contribution to the Kubeflow community effort for Helm-based installation
+---
 
-### Skills Required:
+### Project 4: Standalone Helm Charts
 
-✔️ Experience with Kubernetes & Helm  
-✔️ Understanding of Kubeflow architecture  
-✔️ Familiarity with Helm templating (`values.yaml`, `Chart.yaml`)  
-✔️ YAML & Kubernetes manifests expertise
+__Components:__ Kubeflow Trainer, Kubeflow Katib, Kubeflow Spark Operator, Kubeflow Model Registry
 
-### Nice to Have:
+__Possible Mentors:__ [`@juliusvonkohout`](https://github.com/juliusvonkohout), [`@varodrig`](https://github.com/varodrig)
 
-➕ Experience with KServe & Knative  
-➕ Knowledge of RBAC, authentication & security policies  
-➕ Understanding of Kubernetes networking & ingress controllers
+__Difficulty:__ Medium
 
-#### Mentors:
-- Chase Cristensen
-- Valentina Rodriguez Sosa
-- Julius von Kohout
+__Size:__ 350 hours
 
-#### Resources & References:
+__Goals:__
 
-* Kubeflow Helm Chart Effort: [GitHub Repo](https://github.com/kromanow94/kubeflow-manifests)  
-* Kubeflow Documentation: [Kubeflow.org](https://www.kubeflow.org/)  
-* Helm Documentation: Helm.sh
+- Work with Kubeflow components maintainers to support the creation of Helm charts for some Standalone Kubeflow components.
+- Investigate possible systems to automatically generate or maintain charts based on the existing kustomize manifests.
```

Comment:
Should we say design and implement synchronization script between Helm and Kustomize to keep manifests up-to-date?
For example, @ChenYi015 already implemented one script for KFTrainer: https://github.com/kubeflow/trainer/blob/c0b1d7d26e7a7caea1ffd8b3111b79be6de24b78/hack/sync-manifests.sh